scripts/s3/s3api/mix_object_crud_operations.py

```diff
@@ -199,3 +202,37 @@ def get_sample_details(self, file_size: int) -> tuple:
         else:
             self.log.info(sample_msg, self.delete_samples, "delete")
         return self.write_samples, self.read_samples, self.delete_samples
+
+    def execute_object_crud_workload(self):
+        """Execute Plain object operations workload  for given distribution for specific duration."""
+        distribution = self.distribution
+        while True:
+            crud_iter = 0
```

Comment:
Better to use self.iteration instead of crud_iter as this will be reset again to 0.